src/lib/transition_frontier_persistence/diff_mutant.ml

```diff
@@ -41,16 +41,18 @@ end = struct
 
   module Key = struct
     module New_frontier = struct
+      (* TODO: version *)
```

Comment:
Without fully versioning these types, could we use `V1` instead of `Latest`, for now?